.github/workflows/ci.yml

```diff
@@ -33,15 +30,11 @@ jobs:
         ghc-version: ${{ matrix.ghc }}
     - name: Get GHC libdir
       id: get-ghc-libdir
-      run: |
-        echo "libdir=$(ghc --print-libdir)" >> $GITHUB_OUTPUT
+      run: echo "libdir=$(ghc --print-libdir)" >> $GITHUB_OUTPUT
       shell: bash
     - run: cabal v2-freeze --enable-tests
     - uses: actions/cache@v2
       with:
         path: ${{ steps.setup-haskell.outputs.cabal-store }}
         key: ${{ runner.os }}-${{ matrix.ghc }}-${{ steps.get-ghc-libdir.outputs.libdir }}-${{ hashFiles('cabal.project.freeze') }}
-    - run: cabal new-install apply-refact --install-method=copy
-    - uses: ndmitchell/neil@master
-      with:
-        hlint-arguments: src
+    - run: cabal run exe:hlint -- --test
```

Comment:
Looks good; we can do this temporarily until @ndmitchell fixes it properly. 

But can you still install `apply-refact` for 9.8 and 9.6, so that refactoring tests are run? Skipping all refactoring tests is a bit drastic.

Reply:
sure!